File: market/test/unit/java/ru/yandex/market/logistics/yard/base/SoftAssertionSupport.java
package ru.yandex.market.logistics.yard.base;

import org.assertj.core.api.SoftAssertions;
import org.junit.jupiter.api.AfterEach;

public abstract class SoftAssertionSupport {
    protected final SoftAssertions softly = new SoftAssertions();

    @AfterEach
    public final void triggerSoftAssertions() {
        softly.assertAll();
    }
}

File: market/test/unit/kotlin/base/SoftAssertionSupport.kt
package ru.yandex.market.logistics.calendaring.base

import org.assertj.core.api.SoftAssertions
import org.junit.jupiter.api.AfterEach

abstract class SoftAssertionSupport {
    protected val softly = SoftAssertions()

    @AfterEach
    fun triggerSoftAssertions() {
        softly.assertAll()
    }
}
